### ISSUE TITLE
Unable to build Gazebo8 on macOS Mojave

### ISSUE THREAD

**Issue by Carpetfizz** [osrf/homebrew-simulation#747]
macOS Mojave 10.14.15

[logs](https://gist.github.com/Carpetfizz/8b7a54123d5565a461918c4ad369f51e)

**Comment by scpeters** [comment]
As I said in #656, gazebo8 is now end-of-life, so we aren't proactively fixing issues like this when it fails to build. I would be happy to review a pull request that fixes the build if you can identify a fix. Looking at the build log, I'm wondering if you have `qt@4` installed? I think you need to remove or `brew unlink qt@4` in order to build successfully against `qt5`.

**Comment by Carpetfizz** [comment]
Thanks, will give the unlink a shot and update here if it worked. Just curious, what does end-of-life mean? Is the entire Gazebo project dead/unsupported, or is there a new version of Gazebo? I just followed the installation instructions on the website.

**Comment by scpeters** [comment]
It just means gazebo8 is getting old; I recommend `brew install gazebo9` or `brew install gazebo10`. I'll update the tutorials to reflect this; thanks for the heads-up!

**Comment by Carpetfizz** [comment]
Got it thanks! So I was actually able to get it to work after your recommendation! Is there an easy way for me to update gazebo8 to gazebo10?

**Comment by scpeters** [comment]
they are separate formula that can't be linked at the same time, but you can `brew unlink gazebo8; brew install gazebo10`, which should install gazebo10 from a bottle instead of compiling it from source.

**Comment by Carpetfizz** [comment]
Thanks again for the suggestion, upon doing that it installs fine, but I get the following error when I try to run `gazebo`

```
dyld: Library not loaded: /usr/local/opt/dartsim/lib/libdart-collision-bullet.6.8.dylib
  Referenced from: /usr/local/bin/gazebo
  Reason: image not found
Abort trap: 6
```

**Comment by scpeters** [comment]
Ok, that is an unrelated issue that the bottles need to be rebuilt. I'm working on that in #749

**Comment by scpeters** [comment]
The gazebo9 bottle should be fixed right now, but gazebo10 requires an extra patch:

* https://bitbucket.org/osrf/gazebo/pull-requests/3106/fix-gazebo10-build-with-dart-69/diff

**Comment by Carpetfizz** [comment]
Thanks! I just ended up installing gazebo9 and it seems to work fine. Will keep an eye on the linked PR so I can install gazebo10 when it's fixed.

Really appreciate the help as a first time user!

**Comment by scpeters** [comment]
:tada:

**Comment by scpeters** [comment]
gazebo10 bottle should be fixed now that #750 is merged

**Comment by Carpetfizz** [comment]
Perfect, that worked great. Thank you so much! Looking forward to learning how to use it.
